colibre/scripts/birth_density_distribution.py

```diff
@@ -110,7 +110,7 @@ def critical_density_DVS2012(
 
 for color, (snapshot, name) in enumerate(zip(data, names)):
 
-    birth_densities = (snapshot.stars.birth_densities / mh).to(birth_density_bins.units)
+    birth_densities = snapshot.stars.birth_densities.to("g/cm**3") / mh.to("g")
```

Comment:
The former version of the unit conversion would make densities at n \gtrsim 2e5 H/cm3 infinite, where infinite means that python would store them as `inf` (I guess due to overflow or something?)